### PR TITLE
Release v11.1.0

### PR DESCRIPTION
**Pull request by teich-stripe**
- [x] Update README (if necessary)
- [x] Update CHANGELOG with any new features or breaking changes
- [x] Update MIGRATING (if necessary)
- [x] Update STPSDKVersion in STPAPIClient.h
- [x] Update version in Info.plist (make sure to not change the "InfoDictionary" field)
- [x] Update VERSION
- [x] Update `Stripe.podspec` (make sure there aren't any new directories added, and if so, add them to the "files" section
- [x] Update documentation: `./ci_scripts/build_documentation.sh`
- [x] Make sure `pod lib lint` succeeds without warnings or errors
- [x] Make sure `./ci_scripts/export_builds.sh` succeeds without warnings or errors

### CODE REVIEW

File: docs/docs/Classes.html [hunk 1]
```diff
@@ -938,51 +941,6 @@
                 <li class="item">
                   <div>
                     <code>
-                    <a name="/c:objc(cs)STPCustomerContext"></a>
```
Comment (joeydong-stripe) [inline docs/docs/Classes.html]: Did we lose the docs for `STPCustomerContext`?

Comment (teich-stripe) [inline docs/docs/Classes.html]: Hmm. I just ran the scripts. Why would it get removed?

**Comment by joeydong-stripe** [comment]
Resolved missing `STPCustomerContext` documentation in https://github.com/stripe/stripe-ios/pull/736

Moving forward, we're going to pay more attention to the `docs/docs/undocumented.json` file 😄

**Comment by teich-stripe** [comment]
Excellent plan!

**Pull request merged**